reconcile/quay_membership.py

```diff
@@ -42,6 +42,23 @@
 }
 """
 
+PERMISSIONS_QUERY = """
```

Comment:
https://issues.redhat.com/browse/APPSRE-5962?focusedCommentId=20718097&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20718097

should we avoid using more of the query generator until query collision is solved?

also, for this integration, it would mean a MAJOR refactor IMO.

Reply:
If its a major refactor, then I agree to skip it :+1: 

Concerning collision: in its current state qenerate already resolves collisions by prefixing https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/design-docs/custom-python-code-generator-for-gql-queries.md#class-name-collisions - so in that sense query collision is already solved.

The ticket above is more dedicated to allowing multiple collision strategies (if desired).